cassandra-four-zero/src/main/java/org/apache/cassandra/bridge/SSTableWriterImplementation.java

```diff
@@ -81,17 +85,23 @@ static CQLSSTableWriter.Builder configureBuilder(String inDirectory,
                                                      String createStatement,
                                                      String insertStatement,
                                                      int bufferSizeMB,
+                                                     Set<String> udts,
                                                      IPartitioner cassPartitioner)
     {
-        return CQLSSTableWriter
-               .builder()
-               .inDirectory(inDirectory)
-               .forTable(createStatement)
-               .withPartitioner(cassPartitioner)
-               .using(insertStatement)
-               // The data frame to write is always sorted,
-               // see org.apache.cassandra.spark.bulkwriter.CassandraBulkSourceRelation.insert
-               .sorted()
-               .withMaxSSTableSizeInMiB(bufferSizeMB);
+        CQLSSTableWriter.Builder builder = CQLSSTableWriter
+                                           .builder()
+                                           .inDirectory(inDirectory)
+                                           .forTable(createStatement)
+                                           .withPartitioner(cassPartitioner)
+                                           .using(insertStatement)
+                                           // The data frame to write is always sorted,
+                                           // see org.apache.cassandra.spark.bulkwriter.CassandraBulkSourceRelation.insert
+                                           .sorted()
+                                           .withMaxSSTableSizeInMiB(bufferSizeMB);
+        for (String udt : udts)
+        {
+            builder.withType(udt);
+        }
+        return builder;
```

Comment:
nit
```suggestion
        CQLSSTableWriter.Builder builder = CQLSSTableWriter.builder();
        for (String udt : udts)
        {
            builder.withType(udt);
        }
        
        return builder
               .inDirectory(inDirectory)
               .forTable(createStatement)
               .withPartitioner(cassPartitioner)
               .using(insertStatement)
               // The data frame to write is always sorted,
               // see org.apache.cassandra.spark.bulkwriter.CassandraBulkSourceRelation.insert
               .sorted()
               .withMaxSSTableSizeInMiB(bufferSizeMB);
```

Reply:
I'm not sure why this is any different/better than the existing code - just less indented?